app/build.gradle

```diff
@@ -23,18 +23,18 @@ android {
 }
 
 dependencies {
-    testCompile 'junit:junit:4.12'
+    testImplementation 'junit:junit:4.12'
```

Comment:
I'm not familiar with this, but guessing it's a language change. Is this the result of using a new version of Gradle? Unless something demands it, I would not attempt to deal with switching Gradle versions at the same time as changing SDK version.